src/api/java/CMakeLists.txt

```diff
@@ -58,6 +58,24 @@ add_custom_command(
 )
 add_custom_target(generate-java-proofrules DEPENDS ${JAVA_PROOF_RULE_FILE})
 
+# Generate SkolemFunId.java
+set(JAVA_SKOLEM_FUN_ID_FILE
```

Comment:
This file needs to be added to the list `JAVA_FILES` defined here
https://github.com/cvc5/cvc5/blob/main/src/api/java/CMakeLists.txt#L91. 
Otherwise, the jar file would not include this class. 
The CI should capture this, which implies the unit tests somehow finds the class outside the jar file. 
We need to update the CI in future PR to use the jar file instead of compiled files.

Reply:
Should we add `ProofRule.java` here too?

Reply:
Yes. According to the documentation here 
https://cmake.org/cmake/help/latest/module/UseJava.html. Currently the jar file includes ProofRule.java may be because it is referenced by other classes that are in the jar file. To be safe, we need to add it to the jar file as well.